eras/alonzo/impl/src/Cardano/Ledger/Alonzo/UTxO.hs

```diff
@@ -152,22 +146,11 @@ getAlonzoScriptsNeeded (UTxO u) txBody =
       where
         withdrawals = unWithdrawals $ txBody ^. withdrawalsTxBodyL
         fromRwd accnt = do
-          hash <- scriptCred $ getRwdCred accnt
+          hash <- credScriptHash $ getRwdCred accnt
```

Comment:
how about `scriptCredHash`? since it is the hash of a script credential.

Reply:
That's right, that's why `credScriptHash` is better, because it is a `ScriptHash`, not a `CredHash` :smile: 

Also this way it is consistent with `credKeyHashWitness`, which is quite similar, but for the `KeyHash`es

Reply:
:laughing: yep, makes sense now.